message-bus-kafka/swarm.sh

```diff
@@ -19,34 +18,22 @@ function init_vars() {
 
   UTILS_PATH="${COMPOSE_FILE_PATH}/../utils"
 
-  ZOOKEEPER_SERVICES=(
-    "zookeeper-1"
-  )
-
   UTILS_SERVICES=(
     "kafdrop"
     "kafka-minion"
   )
 
-  if [[ "${CLUSTERED_MODE}" == "true" ]]; then
-    ZOOKEEPER_SERVICES=(
-      "${ZOOKEEPER_SERVICES[@]}"
-      "zookeeper-2"
-      "zookeeper-3"
-    )
-  fi
-
   SERVICE_NAMES=(
-    "${ZOOKEEPER_SERVICES[@]}"
     "${UTILS_SERVICES[@]}"
-    "kafka"
+    "kafka-01"
+    "kafka-02"
+    "kafka-03"
```

Comment:
I think we can put that in a variable alone KAFKA_SERVICES and we can use it later for deploy_sanity and service_reachable function as well